Formula/p/podman-tui.rb

```diff
@@ -0,0 +1,26 @@
+class PodmanTui < Formula
+  desc "Podman Terminal User Interface"
+  homepage "https://github.com/containers/podman-tui"
+  url "https://github.com/containers/podman-tui.git",
+      tag:      "v0.17.0",
+      revision: "b4350927babdaa35fbf50424190683aeb9f5f66f"
+  license all_of: ["Apache-2.0", "GPL-3.0-or-later"]
+  head "https://github.com/containers/podman-tui.git", branch: "main"
+
+  depends_on "go" => :build
+  depends_on "make" => :build
+
+  def install
+    if OS.mac?
+      system "make", "binary-darwin"
+      bin.install "bin/darwin/podman-tui" => "podman-tui"
+    else
+      system "make", "binary"
+      bin.install "bin/podman-tui" => "podman-tui"
+    end
+  end
+
+  test do
+    assert_match "podman-tui v#{version}", shell_output("#{bin}/podman-tui version")
```

Comment:
if not challenging, probably can do a app startup check.

Reply:
Its a bit challening, I can start the tui application in the background and check the logs if its started, however its not easy to kill it after startup since the signals are ignored by the tui.